+++ src/lib/fuse-ext2/e2fsprogs-1.41.12.newgit/ext2fs/unix_io.c
@@ -200,7 +200,7 @@
 	errcode_t	retval;
 	size_t		size, alignsize, fragment;
 	ext2_loff_t	location;
-	int		total = 0, actual;
+	int		total = 0, actual = 0;
 #define BLOCKALIGN 512
 	char		sector[BLOCKALIGN];
 
